### PR TITLE
docs(tabs): add demo tabs select event

### PR DESCRIPTION
**Pull request by ghost**
add demo tabs select method  ( http://joxi.ru/Vm6ZxbeHxl1jYm) related to #3664

### CODE REVIEW

File: demo/src/app/components/+tabs/demos/select-event/select-event.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import { TabDirective } from 'ngx-bootstrap/tabs';
+
+@Component({
+  selector: 'demo-tabs-select-event',
+  templateUrl: './select-event.html'
+})
+export class DemoTabsSelectEventComponent {
+  value: string;
+  onSelect(data: TabDirective): void {
+    this.value = data.elementRef.nativeElement.innerHTML;
```
Comment (IlyaSurmay) [inline demo/src/app/components/+tabs/demos/select-event/select-event.ts]: ?? 
why would you need to access DOM here?

File: demo/src/app/components/+tabs/tabs-section.list.ts [hunk 2]
```diff
@@ -81,6 +82,14 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/styling/styling.html'),
         outlet: DemoTabsStylingComponent
       },
+      {
+        title: 'Select event',
+        anchor: 'select-event',
+        component: require('!!raw-loader?lang=typescript!./demos/select-event/select-event'),
+        html: require('!!raw-loader?lang=markup!./demos/select-event/select-event.html'),
+        description: '<p>You can subscribe to datepicker\'s <code>select</code> event</p>',
```
Comment (IlyaSurmay) [inline demo/src/app/components/+tabs/tabs-section.list.ts]: maybe `tab's`?

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]
commit and PR should be named according to our styleguide. This PR's name should be `docs(tabs): add demo for select event`

File: demo/src/app/components/+tabs/demos/select-event/select-event.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import { TabDirective } from 'ngx-bootstrap/tabs';
+
+@Component({
+  selector: 'demo-tabs-select-event',
+  templateUrl: './select-event.html'
+})
+export class DemoTabsSelectEventComponent {
+  value: string;
+  onSelect(data: TabDirective): void {
+    this.value = (data.heading).slice(6);
```
Comment (IlyaSurmay) [inline demo/src/app/components/+tabs/demos/select-event/select-event.ts]: don't manipulate the string at all or use `replace`

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

File: demo/src/app/components/+tabs/demos/select-event/select-event.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<pre *ngIf="value">Event select is fired. Your choice is: <strong>{{value}}</strong></pre>
```
Comment (EvilAlexei) [inline demo/src/app/components/+tabs/demos/select-event/select-event.html]: Add classes to `pre`, to be more consistent.
`card card-block card-header`

**Review by EvilAlexei — CHANGES_REQUESTED** [review]
Change header according to commit style guide

**Comment by ghost** [comment]
update and testing my branch according to Yevheniia comments(conflicts in files) which I can't see

File: demo/src/app/components/+tabs/demos/select-event/select-event.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<div class="mb-3">
+  <pre class="card card-block card-header" *ngIf="value">Event select is fired. Your choice is: {{value}}</pre>
+</div>
+<tabset>
+  <tab heading="04.05.2018" (select)="onSelect($event)">04.05.2018</tab>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+tabs/demos/select-event/select-event.html]: remove these dates and replace them with headings like `First tab` and `Second tab`

File: demo/src/app/components/+tabs/demos/select-event/select-event.html [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<div class="mb-3">
+  <pre class="card card-block card-header" *ngIf="value">Event select is fired. Your choice is: {{value}}</pre>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+tabs/demos/select-event/select-event.html]: change message to `Event select is fired. The heading of the selected tab is: `

File: demo/src/app/components/+tabs/demos/select-event/select-event.html (hunk 5 above)
Comment (IlyaSurmay) [reply demo/src/app/components/+tabs/demos/select-event/select-event.html]: also, change tab content to something more generic like here https://github.com/valor-software/ngx-bootstrap/pull/3652/files#diff-cf724fc338926969507b9d22f7ce773f

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

**Review by EvilAlexei — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — APPROVED** [review]

**Pull request merged**